scripts/postversion.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I thought there was an "rm -r tmp" command in there, not?  Even so, that may be unnecessary because I added tmp/ to .gitignore.